### PR TITLE
ROC-3408: update response check and send page

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3408

### Change description
Refactor response check and send page

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [x] New pages included in a11y tests
- [x] Task list and task completeness checks updated
- [x] Check and send page updated
- [x] Routes, page content and page flows of new features are toggled 
- [x] UI changes look good on mobile
- [x] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/claim/views/check-and-send.njk [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 {% from "table.njk" import tableStart, tableEnd, row %}
 {% from "form.njk" import csrfProtection, submitButton, errorSummary %}
-{% from "../../../views/macro/statementOfTruth.njk" import statementOfTruth %}
+{% from "statementOfTruth.njk" import statementOfTruth %}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: I think this is using the wrong statementOfTruth macro, I have a feeling that rafal copied it into the app at that location

File: src/main/features/response/views/check-and-send.njk [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {% extends "layout.njk" %}
 
-{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+{% from "table.njk" import tableStart, tableEnd, row %}
 {% from "form.njk" import csrfProtection, submitButton, errorSummary %}
-{% from "../../../views/macro/statementOfTruth.njk" import statementOfTruth %}
+{% from "statementOfTruth.njk" import statementOfTruth %}
```
Comment (timja-kainos) [inline src/main/features/response/views/check-and-send.njk]: same here

File: src/main/features/claim/views/check-and-send.njk [hunk 3]
```diff
@@ -74,21 +76,20 @@
...
-    {{ row('Your evidence (optional):', '', paths.evidencePage.uri, false ) }}
-    {% set allEvidenceRows = draftClaim.evidence.getPopulatedRowsOnly() %}
-    {% set lastItemIndex = allEvidenceRows.length - 1 %}
-    {% for evidenceRow in allEvidenceRows %}
-      {{ row(evidenceRow.type.displayValue, evidenceRow.description | default(''), '', loop.index0 === lastItemIndex) }}
-    {% endfor %}
-
+    {{
+      timelineCheckAndSend(
+        header = 'Timeline of what happened',
```
Comment (geek-soft) [inline src/main/features/claim/views/check-and-send.njk]: No colon at the end? There are still some places with colon so I wonder what is the approach. The same in other refactored places.

File: src/main/views/macro/evidenceCheckAndSend.njk [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% from "table.njk" import row %}
+
+{% macro evidenceCheckAndSend(header, evidence, url) %}
+
+  {{ row(label = header, value = '', changeLink = url, bottomBorder = false, bold = true) }}
+  {% set lastItemIndex = evidence.length - 1 %}
+  {% for evidenceRow in evidence %}
+    {{ row(evidenceRow.type.displayValue, evidenceRow.description | default(''), '', loop.index0 === lastItemIndex) }}
```
Comment (geek-soft) [inline src/main/views/macro/evidenceCheckAndSend.njk]: Would be great to have `label =`, `value = `, `changeLink = ` and `bold = ` here as well, especially to highlight what `loop.index0 === lastItemIndex` is used for.

File: src/main/views/macro/timelineCheckAndSend.njk [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{% from "table.njk" import row %}
+
+{% macro timelineCheckAndSend(header, timeline, url) %}
+
+  {{ row(label = header, value = '', changeLink = url, bottomBorder = false, bold = true) }}
+  {% set lastItemIndex = timeline.length - 1 %}
+  {% for timelineRow in timeline %}
+    {{ row(timelineRow.date, timelineRow.description, '', loop.index0 === lastItemIndex) }}
```
Comment (geek-soft) [inline src/main/views/macro/timelineCheckAndSend.njk]: The same here, named arguments would make it more readable.

File: src/main/features/claim/views/check-and-send.njk (hunk 3 above)
Comment (yarpo) [reply src/main/features/claim/views/check-and-send.njk]: No, all colons should be removed from check and send pages according to latest pattern

**Pull request merged**